### PR TITLE
Adding shridharkapshikar file

### PR DESCRIPTION
**Pull request by shridharkapshikar**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1374"></a>

### CODE REVIEW

File: public/data/shridharkapshikar.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (emmalearnscode) [inline public/data/shridharkapshikar.json:1]: You are missing an opening { on line 1

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Your profile looks good apart from the missing curly brace at the beginning. If you add that in then this should be ready to merge.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: public/data/shridharkapshikar.json (hunk 1 above)
Comment (shridharkapshikar) [reply public/data/shridharkapshikar.json]: Hi Team, I have corrected the missing { on line 1. Request you to review & do the needful.

File: public/data/shridharkapshikar.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "June 2022",
+      "icon": "dollar",
+      "color": "Orange",
+      "description": "Started freelancing",
```
Comment (emmalearnscode) [inline public/data/shridharkapshikar.json:24]: You now have a trailing comma on line 24 which is causing a couple of tests to fail. Deleting the comma should fix the issue.
```suggestion
      "description": "Started freelancing"
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Comment (shridharkapshikar) [reply public/data/shridharkapshikar.json]: Sorry for typo. 
I've the made the correction.

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by emmalearnscode** [comment]
@shridharkapshikar the page is not automatically deployed when profiles are added which is why it’s not there yet. I’ll trigger a manual deployment now so it should appear shortly

**Pull request merged**